content/en/docs/release.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Release schedule and support policy"
+description: "This document describes the Notary Project release schedule and support policy"
+type: docs
+weight: 10
+---
+
+Since the Notary Project v1.0.0 release, Notary Project releases are scheduled quarterly.
+
+## Release versioning
+
+The Notary Project release version is in the form of `vX.Y.Z`, where `X` is the major version, `Y` is the minor version and `Z` is the patch version, following [Semantic Versioning](https://semver.org/spec/v2.0.0.html) terminology.
+
+### Patch release
+
+Patch releases provide users with bug fixes and security fixes. They do not contain new features. A patch release can be done by each Notary Project repository individually every quarter in a calendar year. A patch release to fix critical issues or security issues can be done on demand. A patch release can be cancelled if there is no new fixes since the previous release.
```

Comment:
```suggestion
Patch releases provide users with bug fixes and security fixes. They do not contain new features. A patch release can be done by each Notary Project repository individually every quarter in a calendar year. A patch release to fix critical issues or security issues can be done on demand. ~~A patch release can be cancelled if there is no new fixes since the previous release.~~
```
The last line is not needed. It is more confusing to say "Patch release can be cancelled" then not to say anything. It is obvious that if no fixes are there, no patch release will be created. The cancelling word may imply that we may do a patch release but can redact it later.